### ISSUE TITLE
Attempting to call `parentInstance.removeChild` when `parentInstance.removeChild` is not defined

### ISSUE THREAD

**Issue by bkoltai** [diegomura/react-pdf#182]
I'm using `@react-pdf/dom` and am showing the PDF iframe in a modal when the user clicks a button. When I unmount the modal, I'm getting the following error

```
renderer.js:92 Uncaught TypeError: parentInstance.removeChild is not a function
    at removeChildFromContainer (renderer.js:92)
    at unmountHostComponents (react-reconciler.development.js:4974)
    at commitDeletion (react-reconciler.development.js:5020)
    at commitAllHostEffects (react-reconciler.development.js:5712)
    at HTMLUnknownElement.callCallback (react-reconciler.development.js:1613)
    at Object.invokeGuardedCallbackDev (react-reconciler.development.js:1652)
    at invokeGuardedCallback (react-reconciler.development.js:1509)
    at commitRoot (react-reconciler.development.js:5801)
    at performWorkOnRoot (react-reconciler.development.js:6803)
    at performWork (react-reconciler.development.js:6753)
```

I've modified the `dom-bindings` example to demonstrate this issue [here](https://github.com/bkoltai/react-pdf/tree/broken-dom-unmounting).

You can reproduce the error by cloning the repo and running the `dom-bindings` example as normal. Click the "toggle" button to see the error.

**Comment by bkoltai** [comment]
I've followed the code to the best of my ability and I think the issue is that the render ultimately calls `removeChildFromContainer` with a `parentContainer` as an instance of `PDFDocument`. 

I believe the fix is to simply check that the `parentInstance.removeChild` function exists before calling it, though admittedly, I'd don't fully understand the implications of that.

It may cause the `PDFDocument` to not get destructed in a way it is intended to be.

I've created a PR to make the appropriate change if you think it's the right approach.
